src/config.c

```diff
@@ -198,6 +198,9 @@ typedef enum numericType {
     NUMERIC_TYPE_TIME_T,
 } numericType;
 
+#define NUM_CONF_FLAG_MEMORY (1<<0) /* Indicates if this value can be loaded as a memory value */
+#define NUM_CONF_FLAG_PERCENT (1<<1) /* Indicates if this value can be loaded as a percent (and stored as a negative int) */
```

Comment:
i suggest to keep INTEGER_CONFIG and MEMORY_CONFIG (don't rename them),
so you don't end up modifying all the numeric config lines (unnecessary conflicts with other PRs)

Reply:
:+1: 
Fixed.